### PR TITLE
[GT-181] Add last logged in field for admins

### PR DESCRIPTION
**Pull request by Sae126V**
[Functionality]

To help with user support when someone queries one of the account deletion emails, it would be useful to be able to see when the account was last logged in to in the web interface.

"Resolves https://github.com/GOCDB/gocdb/issues/456 ".

### CODE REVIEW

**Comment by Sae126V** [comment]
Ready to be reviewed.

File: htdocs/web_portal/views/user/view_user.php [hunk 1]
```diff
@@ -127,6 +129,21 @@
                         </td>
                     </tr>
                 <?php } ?>
+                <?php if ($isCallingUserAdmin) : ?>
+                <tr class="site_table_row_1">
+                    <td class="site_table">Last Logged In</td>
+                    <td class="site_table">
+                        <?php
+                        if ($params['user']->getLastLoginDate()) {
+                            xecho(
```
Comment (rowan04) [inline htdocs/web_portal/views/user/view_user.php:138]: what is xecho?

Comment (Sae126V) [reply htdocs/web_portal/views/user/view_user.php:138]: It's a function to mark and say it is safe content @rowan04 . Please have a look at this file htdocs/web_portal/index.php. It has nice link attached to the function.

`function xecho($data) {
   echo xssafe($data);
}`

**Comment by gregcorbett** [comment]
Looks good to me. 

There's a slight issue with the colour of the rows if a User doesn't have a Home Site (I forgot this was a thing and it is being removed from GOCDB #266), however this is probably easiest to fixed as part of the wider refatoring to how the colour of the rows is decided.

image

vs

image

I've opened #478  to track this.

**Review by gregcorbett — APPROVED** [review]

**Comment by gregcorbett** [comment]
rebasing on top of latest `dev`

**Pull request merged**